charts/dev/livebook/values.yaml

```diff
@@ -50,3 +51,20 @@ workload:
               secretKeyRef:
                 name: secrets
                 key: LIVEBOOK_COOKIE
+            LIVEBOOK_PASSWORD:
+              secretKeyRef:
+                name: secrets
+                key: LIVEBOOK_PASSWORD
+            LIVEBOOK_DEBUG: '{{ ternary "true" "false" .Values.livebook.debug }}'
+            LIVEBOOK_UPDATE_INSTRUCTIONS_URL: "{{ .Values.livebook.updateInstructionsUrl }}"
+            LIVEBOOK_AWS_CREDENTIALS: '{{ ternary "true" "false" .Values.livebook.awsCredentials.enabled }}'
+            AWS_ACCESS_KEY_ID: "{{ .Values.livebook.awsCredentials.accessKeyId }}"
+            AWS_SECRET_ACCESS_KEY:
+              secretKeyRef:
+                name: secrets
+                key: AWS_SECRET_ACCESS_KEY
+# default values
+livebook:
+  debug: false
+  awsCredentials:
+    enabled: false
```

Comment:
please add all the fields here, like password etc. same defaults as in questins.yaml.

Someone that does not use scale would not know what to change or where to put values

Reply:
Now I don't get you. The whole point was for password not to have a default value in the first place. Also the others don't have a default value in questions.yaml. They're optional...

Reply:
yes you add it here like this `password: ""`
same for the other options you have added in the questions.yaml

questions.yaml is not something normal helm users use or know about.
So options have to be also in values.yaml

Reply:
OK sure. The problem I have with that is that now you can `helm install` or `helm template` the helm chart without additional input. You're not forced to define your values. It will only fail at runtime (because the password is not long enough etc...). I don't really like this behaviour. But your decision.

Reply:
@stavros-k wdyt?

Reply:
This is fine, people should do a basic, "what I am installing", and "what values can I supply".

I'd argue that an error coming from helm or k8s is gonna be intepreted as a chart bug, instead of an missconfiguration error.

You can add a custom message in the secret, like `.Values....pass | required "my message" | quote`

I'll merge this now, but feel free to do so in another PR (eg when moving trains or fixing something else)